(dp1
S'output'
p2
S'<type \'exceptions.SyntaxError\'> Field: invalid table name: qwertyuiop[, use rname for "funny" names'
p3
sS'layer'
p4
S'/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/applications/lifeTracker/controllers/api.py'
p5
sS'code'
p6
S'<code object <module> at 0x109f5c130, file "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/applications/lifeTracker/controllers/api.py", line 1>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/applications/lifeTracker/controllers/api.py", line 97, in <module>\n  File "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/gluon/globals.py", line 421, in <lambda>\n    self._caller = lambda f: f()\n  File "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/gluon/tools.py", line 3867, in f\n    return action(*a, **b)\n  File "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/applications/lifeTracker/controllers/api.py", line 46, in create_table\n    Field(\'time\', \'datetime\', update=get_current_time())\n  File "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/gluon/packages/dal/pydal/base.py", line 590, in define_table\n    table = self.lazy_define_table(tablename, *fields, **kwargs)\n  File "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/gluon/packages/dal/pydal/base.py", line 602, in lazy_define_table\n    table = table_class(self, tablename, *fields, **kwargs)\n  File "/Users/ethantom/Documents/School/CMPS 183/life-tracker/web2py/gluon/packages/dal/pydal/objects.py", line 241, in __init__\n    \'use rname for "funny" names\' % tablename)\nSyntaxError: Field: invalid table name: qwertyuiop[, use rname for "funny" names\n'
p11
s.